L1Trigger/L1TTwinMux/src/DTRPCBxCorrection.cc

```diff
@@ -180,15 +199,16 @@ std::vector<int> DTRPCBxCorrection::concat_delta(vector<int> delta_0, vector<int
 }
 
 int DTRPCBxCorrection::sign(float inv){
-	if(inv<0) return -1;
-	if(inv>0) return 1;
-	return 0;
+  if(inv<0) return -1;
+  if(inv>0) return 1;
+  return 0;
 }
 
 int DTRPCBxCorrection::noRPCHits(L1MuTMChambPhContainer inCon, int bx, int wh, int sec, int st){
   int size = 0;
   const std::vector<L1MuDTChambPhDigi>* vInCon = inCon.getContainer();
   for ( std::vector<L1MuDTChambPhDigi>::const_iterator i = vInCon->begin(); i != vInCon->end(); i++ ) {
+  //for ( auto &i:vInCon){
```

Comment:
Does this not work?
`for ( const auto &i : vInCon) {`

Reply:
no it crashes with  error: no matching function for call to 'begin(const std::vector<L1MuDTChambPhDigi>*&)

Reply:
Ah I see that vInCon is a pointer. Try `for ( const auto &i : *vInCon) {`